atm.js

```diff
@@ -1 +1,27 @@
-// your code goes here!
+var deposit1 = document.querySelector ("#checkingDepositButton")
+var withdraw1 = document.querySelector ("#checkingWithdrawalButton")
+var deposit2 = document.querySelector ("#savingsDepositButton")
+var withdraw2 = document.querySelector ("#savingsWithdrawalButton")
+
+function handleClickEvent (){
+
+  console.log("I was clicked");
+  console.log(this);
+
+}
+// This is the eventListener.
+button.addEventListener("click", handleClickEvent );
```

Comment:
The variable button has not been defined anywhere so this will not work, until it you define it.